### ISSUE TITLE
Contrastive Learning for Improving ASR Robustness in Spoken Language Understanding

### ISSUE THREAD

**Issue by InabaTatsuro** [InabaTatsuro/papers#53]
# Contrastive Learning for Improving ASR Robustness in Spoken Language Understanding [[ICASSP22](https://arxiv.org/abs/2205.00693)]

## Abstract
- improve the robustness of the ASR model by contrastive pretraining
- fine-tuning with self-distillation to reduce the label noises due to ASR errors

### Pretraining
<img width="400" alt="Screenshot 2024-01-10 at 16 24 29" src="https://github.com/InabaTatsuro/papers/assets/102784221/20e721bd-25ab-4164-bfcf-eb41c3310103">

- improve robustness with only textual information
  - previous works need additional speech-related input features

### Fine-tuning
<img width="600" alt="Screenshot 2024-01-10 at 16 32 27" src="https://github.com/InabaTatsuro/papers/assets/102784221/45015a4d-10b3-4323-a18c-292a1010000a">

- hard + soft + self-distillation
- self-distillation is calculated by KL divergence between current and previous prediction
- soft loss is calculated by the similarity
